packages/api-core/src/marketplace-products/fetchers.ts

```diff
@@ -0,0 +1,9 @@
+import { wpcom } from '../wpcom-fetcher';
+import { WooCommercePlugin } from './types';
+
+export function fetchWooCommercePlugin( slug: string ): Promise< WooCommercePlugin > {
```

Comment:
Ohhh.... interesting! Now it starts to make sense. Thanks for investigating it.

But I wonder if it's truly specific to WooCommerce. Hmm. What about other plugin integrations like Sensei (not sure if it still exists)? I wonder if we should just call it `MarketplacePlugin` instead? This existing code call it that way 🤔 

https://github.com/Automattic/wp-calypso/blob/3846728f3e0f9fb23cd4a631de6a6335d653dc16/packages/api-core/src/site-marketplace/mutators.ts#L8

Reply:
Renamed it.